packages/redwood/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vercel/redwood",
-  "version": "0.5.2-canary.3",
+  "version": "1.0.0-canary.2",
```

Comment:
I will make this 1.0.0 when everything is ready to go, unless advised otherwise :)